src/features/flows/nRF52840/index.ts

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import Verify from '../../../common/steps/5xFamilyVerify';
+import Apps from '../../../common/steps/Apps';
+import Develop from '../../../common/steps/develop';
+import Evaluate from '../../../common/steps/Evaluate';
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import { Choice } from '../../../common/steps/program/programSlice';
+import Rename from '../../../common/steps/Rename';
+
+const infoConfig = {
+    title: 'Versatile single-board development kit',
+    markdownContent:
+        '![nRF52840 DK](52840DK.png)  \n&nbsp;  \nThe nRF52840 DK is a versatile single-board development kit for Bluetooth Low Energy, Bluetooth mesh, Thread, Zigbee, 802.15.4, ANT and 2.4 GHz proprietary applications on the nRF52840 SoC. It is the recommended Nordic development kit for Amazon Sidewalk. It also supports development on the nRF52811 SoC.  \n&nbsp;  \n&nbsp;  \n![Technologies](52840DKTech.png)  \n&nbsp;  \nThe nRF52840 DK can also be used for Matter over Thread where Thread is used for transport and Bluetooth LE for commissioning. Matter devices based on Thread are required to feature Bluetooth LE concurrently to enable adding new devices to a network.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ug_nrf52840_dk/page/UG/dk/intro.html)',
```

Comment:
Matter is a separate protocol and needs to be mentioned in the list, even if underneath it uses Matter over Thread.
```suggestion
        '![nRF52840 DK](52840DK.png)  \n&nbsp;  \nThe nRF52840 DK is a versatile single-board development kit for Bluetooth® Low Energy, Bluetooth Mesh, Matter, Thread, Zigbee, 802.15.4, ANT, and 2.4-GHz proprietary applications on the nRF52840 SoC. It is the recommended Nordic development kit for Amazon Sidewalk. It also supports development on the nRF52811 SoC.  \n&nbsp;  \n&nbsp;  \n![Technologies](52840DKTech.png)  \n&nbsp;  \nThe nRF52840 DK can be used for Matter over Thread where Thread is used for transport and Bluetooth LE for commissioning. Matter devices based on Thread are required to feature Bluetooth LE concurrently to enable adding new devices to a network.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ug_nrf52840_dk/page/UG/dk/intro.html)',
```